python/demo/demo_scattering_boundary_conditions/mesh_wire.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2022 Michele Castriotta, Igor Baratta, Jørgen S. Dokken
+
+import sys
+
+try:
+    import gmsh
+except ModuleNotFoundError:
+    print("This demo requires gmsh to be installed")
+    sys.exit(0)
+from numpy import pi
+
+from mpi4py import MPI
+
+
+def generate_mesh_wire(radius_wire, radius_dom, in_wire_size, on_wire_size,
+                       bkg_size, boundary_size, au_tag, bkg_tag, boundary_tag):
+
+    gmsh.initialize(sys.argv)
+    if MPI.COMM_WORLD.rank == 0:
```

Comment:
Would be better to avoid `COMM_WORLD` and let the caller be responsible for calling it on only one rank.